test_planner.cpp

```diff
@@ -15,221 +15,112 @@
  *
 */
 
-#include <rmf_traffic/agv/Planner.hpp>
-#include <rmf_traffic/agv/debug/debug_Planner.hpp>
-#include <rmf_traffic/agv/RouteValidator.hpp>
-
-#include <rmf_traffic/geometry/Circle.hpp>
-
-#include <rmf_traffic/schedule/Database.hpp>
-#include <rmf_traffic/schedule/Participant.hpp>
-
-#include <rmf_fleet_adapter/agv/parse_graph.hpp>
+#include <rmf_performance_tests/rmf_performance_tests.hpp>
 
 #include <iostream>
 
-const std::size_t NotObstacleID = std::numeric_limits<std::size_t>::max();
-
-rmf_traffic::schedule::Participant add_obstacle(
-  const rmf_traffic::agv::Planner& planner,
-  const std::shared_ptr<rmf_traffic::schedule::Database>& database,
-  const rmf_traffic::agv::Plan::Start& start,
-  const rmf_traffic::agv::Plan::Goal& goal)
-{
-  const auto N = database->participant_ids().size();
-
-  auto new_obstacle = rmf_traffic::schedule::make_participant(
-        rmf_traffic::schedule::ParticipantDescription{
-          "obstacle_" + std::to_string(N),
-          "obstacles",
-          rmf_traffic::schedule::ParticipantDescription::Rx::Unresponsive,
-          planner.get_configuration().vehicle_traits().profile()
-        }, database);
-
-  const auto plan = planner.plan(start, goal);
-  new_obstacle.set(plan->get_itinerary());
-  return new_obstacle;
-}
-
-void print_result(
-  const std::string& label,
-  const std::size_t samples,
-  const double total_time,
-  const std::size_t node_count)
+int main(int argc, char* argv[])
 {
-  std::cout << label
-            << "\n -- Total time for " << samples << " samples: "
-            << total_time
-            << "\n -- Average time per run: " << total_time/samples
-            << "\n -- Node count: " << node_count
-            << "\n" << std::endl;
-}
-
-double test_planner_timing_no_cache(
-  const std::string& label,
-  const std::size_t samples,
-  const rmf_traffic::agv::Planner::Configuration& config,
-  const rmf_traffic::agv::Planner::Options& options,
-  const rmf_traffic::agv::Plan::Start& start,
-  const rmf_traffic::agv::Plan::Goal& goal)
-{
-  // Run a test where we produce a new planner every time so we see what the
-  // timing is if the cache is blank
-  const auto begin_time = std::chrono::steady_clock::now();
-  for (std::size_t i=0; i < samples; ++i)
+  if (argc < 2)
   {
-    rmf_traffic::agv::Planner planner(config, options);
-    planner.plan(start, goal);
+    std::cout << "Please provide scenario file name" << std::endl;
+    return 0;
```

Comment:
An exit code of 0 usually implies that the process finished successfully, but in this case the user has given us invalid arguments. We'll usually want to return 1 for general errors.